### ISSUE TITLE
IE 9 issue - Unable to get property 'rail' of undefined or null reference

### ISSUE THREAD

**Issue by kumkanillam** [inuyaksa/jquery.nicescroll#697]
I am using v3.7.6. getting the below error in IE 9

> Unable to get property 'rail' of undefined or null reference

It points to the below line.
   
``` self.ontouchmove = function (e, byiframe) {

            if (!self.rail.drag) return true;

            if (e.targetTouches && opt.preventmultitouchscrolling) {
              if (e.targetTouches.length > 1) return true; // multitouch
            }```

**Comment by ak868308** [comment]
I have the same issue :(

**Comment by nico-villalonga** [comment]
Hi I know this is old, but we are using the library at work and I fix this problem with [pr #796](https://github.com/inuyaksa/jquery.nicescroll/pull/796). If anyone come to this issue, you can download nicescroll (not minified) update changes and minify it to use.
For minification you can use task runners or simply [UglifyJS](https://github.com/mishoo/UglifyJS2)
If you are looking for scrolling libraries take into account this is unmaintained.

**Comment by kumkanillam** [comment]
Now it's occurring for all the browsers with touch enabled. Thank you for the fix @nico-villalonga also please refer this ref #762 its clearly explaining the case.
